hack/image/build-ansible-image.sh

```diff
@@ -12,12 +12,15 @@ BASEIMAGEDIR="$TMPDIR/ansible-operator"
 mkdir -p "$BASEIMAGEDIR"
 go build -o $BASEIMAGEDIR/scaffold-ansible-image ./hack/image/ansible/scaffold-ansible-image.go
 
+# build binary for specific target platform (for purposes of base image only)
+env GOOS=linux GOARCH=amd64 go build -o $BASEIMAGEDIR/ansible-operator-dev-linux-gnu ./cmd/ansible-operator/main.go
+
```

Comment:
We need in the Makefile has a target to just build each binary.

Reply:
Based on https://github.com/operator-framework/operator-sdk/pull/3331#discussion_r449210333, I am leaving this as it is. I think we will be having this specifically in make target after we have a ansible/helm operator makefile in their respective repository.